satosa/logo.png

```diff
@@ -0,0 +1,2 @@
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

Comment:
This looks like an SVG not a PNG (so the file extension should be fixed :sweat_smile:).

Reply:
Oof.  Sorry about that.  Resolved in f31d533.